watson/cli.py

```diff
@@ -136,7 +134,7 @@ def cli(ctx, color):
 
     You just have to tell Watson when you start working on your
     project with the `start` command, and you can stop the timer
-    when you're done with the `stop` command.
+    when yo're done with the `stop` command.
```

Comment:
oops, looks like `sed` found a false-positive :wink: 

```suggestion
    when you're done with the `stop` command.
```